mslib/mscolab/file_manager.py

```diff
@@ -57,43 +58,50 @@ def create_operation(self, path, description, user, last_used=None, content=None
         description: description of the operation
         """
         # set codes on these later
-        if path.find("/") != -1 or path.find("\\") != -1 or (" " in path):
-            logging.debug("malicious request: %s", user)
-            return False
-        proj_available = Operation.query.filter_by(path=path).first()
-        if proj_available is not None:
+        if content is None or verify_waypoint_data(content):
```

Comment:
I think this condition should be inverted into an early return. I.e.:
```
if content is not None and not verify_waypoint_data(content):
    return False
```

The rest of the method could then stay as it was and assume that content is either None or verified, making the diff much smaller and the method more readable.